src/test/regress/output/dispatch.source

```diff
@@ -1073,3 +1073,122 @@ reset gp_print_create_gang_time;
 reset optimizer;
 drop function cleanupAllGangs();
 drop table t_create_gang_time;
+--
+-- On QE backends, GUC options are not actually read from the file, but rather
+-- sent to QEs by QD. It means that default settings on segments have source of
+-- PGC_S_CLIENT instead of PGC_S_FILE, which is higher.
+--
+-- GUCs that were changed in the configuration file of the QD should be re-sent
+-- to segments on gpstop -u, if GUC_GPDB_NEED_SYNC is present if the flags.
+--
+CREATE OR REPLACE FUNCTION show_on_segments(show_name text)
+RETURNS table (guc_name text, guc_setting text)
+EXECUTE ON ALL SEGMENTS AS $$
+BEGIN
+	RETURN QUERY SELECT name, setting FROM pg_settings WHERE show_name = name;
+END;
+$$ LANGUAGE plpgsql;
+-- Explicitly set GUCs through gpconfig, then run gpstop -u.
+--
+-- statment_mem has GUC_GPDB_NEED_SYNC in flags, so it should be synchronized
+-- across the segments. backslash_quote is not synchronized, so it should
+-- remain unchanged on segments.
+SHOW statement_mem;
+ statement_mem 
+---------------
+ 125MB
+(1 row)
+
+SELECT * from show_on_segments('statement_mem');
+   guc_name    | guc_setting 
+---------------+-------------
+ statement_mem | 128000
+ statement_mem | 128000
+ statement_mem | 128000
+(3 rows)
+
+SHOW backslash_quote;
+ backslash_quote 
+-----------------
+ safe_encoding
+(1 row)
+
+SELECT * from show_on_segments('backslash_quote');
+    guc_name     |  guc_setting  
+-----------------+---------------
+ backslash_quote | safe_encoding
+ backslash_quote | safe_encoding
+ backslash_quote | safe_encoding
+(3 rows)
+
+--start_ignore
+\!gpconfig -c statement_mem -v '50MB' -m '100MB'
```

Comment:
The contents of the ignore block are redundant in the output.

Reply:
> There is no output from the function in the input file, so it will be very inconvenient to look at both files to understand which output is correct for which settings. You can look at these ignore blocks as comments which say input values before the function call.
>
> Without these ignore blocks, the output file will look like a comment, a function call, it's results, then another comment, a function call and there's suddenly different results.

These blocks help to preserve the context of what's GUCs are being changed.